performance/aggregate_benchmark_results.rb

```diff
@@ -0,0 +1,80 @@
+# frozen_string_literal: true
+
+require 'json'
+require 'caxlsx'
+require 'fast_excel'
+require 'byebug'
+
+# Common variables
+path_prefix = 'performance/github_artifacts'
+json_folder = "#{path_prefix}/json"
+xlsx_folder = "#{path_prefix}/xlsx"
+
+# Ensure folders exist
+`mkdir -p #{json_folder}`
+`mkdir -p #{xlsx_folder}`
```

Comment:
I'm kind of against using system commands when there is a `Dir` class

Reply:
This should be resolved